man/systemd.netdev.xml

```diff
@@ -1220,7 +1220,19 @@
         <term><varname>DiscoverPathMTU=</varname></term>
         <listitem>
           <para>Takes a boolean. When true, enables Path MTU Discovery on
-          the tunnel.</para>
+          the tunnel.
+          When <varname>IgnoreDontFragment=</varname> is explicitly set to true,
```

Comment:
`is explicitly set to true` -> `is enabled` is simpler??

Reply:
done